guides/govuk-paas.md

```diff
@@ -62,6 +62,119 @@ Instead, transport security is achieved by using HTTPS/TLS and authentication vi
 * [Pricing](https://www.cloud.service.gov.uk/pricing)
 * [Cloud Foundry](https://docs.cloudfoundry.org/)
 
-## Contact
+## Organisations and spaces
+To separate users, applications, services and ultimately costs, paas provides [organisations and spaces](https://docs.cloud.service.gov.uk/orgs_spaces_users.html#managing-organisations-spaces-and-users).
 
-If you are interested in adopting this platform please contact [digital-tools-support@digital.education.gov.uk](mailto:digital-tools-support@digital.education.gov.uk)
+![Organisations and spaces](/assets/images/orgs_and_spaces.png)
+### Organisations
+We create a paas organisation per DfE organisational unit, such as Teacher services, National Retraining Scheme, etc. Therefore each organisational unit may only pay for what they use.
+
+As GDS manage a long list of orgs, organisations are namespaced by department, for example: dfe-teacher-services and dfe-national-retraining-scheme.
```

Comment:
Replace dfe-national-retraining-scheme with a more recent example, National Retraining Scheme has been stopped. So dfe-childrens-social-care is one I think.

Reply:
Yep, agree with Himal that dfe-childrens-social-care can be used as an example...